src/static/riot/competitions/public-list.tag

```diff
@@ -0,0 +1,209 @@
+<public-list>
+    <h1>Public Competitions</h1>
+    <div class="pagination-nav" hide="{(get_array_length(competitions.results) === competitions.count) || (get_array_length(competitions.results) < 10)}">
+        <button show="{competitions.previous}" onclick="{handle_ajax_pages.bind(this, -1)}" class="float-left ui inline button active">Back</button>
+        <button hide="{competitions.previous}" disabled="disabled" class="float-left ui inline button disabled">Back</button>
+        { current_page }
+        <button show="{competitions.next}" onclick="{handle_ajax_pages.bind(this, 1)}" class="float-right ui inline button active">Next</button>
+        <button hide="{competitions.next}" disabled="disabled" class="float-right ui inline button disabled">Next</button>
+    </div>
+    <div each="{competition in competitions.results}">
+        <a class="link-no-deco" href="../{competition.id}">
+            <div class="tile-wrapper">
+                <div class="ui square tiny bordered image img-wrapper">
+                    <img src="{competition.logo}">
+                </div>
+                <div class="comp-info">
+                    <h4 class="heading">
+                        {competition.title}
+                    </h4>
+                    <p class="comp-description">
+                        { pretty_description(competition.description)}
+                    </p>
+                    <p class="organizer">
+                        <em>Organized by: <strong>{competition.created_by}</strong></em>
+                    </p>
+                </div>
+                <div class="comp-stats">
+                    {pretty_date(competition.created_when)}
+                    <div class="ui divider"></div>
+                    <strong>{competition.participant_count}</strong> Participants
+                </div>
+            </div>
+        </a>
+    </div>
+    <div class="pagination-nav" hide="{get_array_length(competitions.results) === competitions.count}">
+        <button show="{competitions.previous}" onclick="{handle_ajax_pages.bind(this, -1)}" class="float-left ui inline button active">Back</button>
+        <button hide="{competitions.previous}" disabled="disabled" class="float-left ui inline button disabled">Back</button>
+        { current_page }
```

Comment:
should we say "page 1 of X" or something here? I saw a "1" floating and it was kinda confusing at first